lib/vtls/openssl.c

```diff
@@ -2749,7 +2749,10 @@ static CURLcode ossl_connect_step1(struct Curl_easy *data,
     /* expects length prefixed preference ordered list of protocols in wire
      * format
      */
-    SSL_CTX_set_alpn_protos(backend->ctx, protocols, cur);
+    if(SSL_CTX_set_alpn_protos(backend->ctx, protocols, cur) != 0) {
```

Comment:
We generally don't compare with `!= 0` in code, so I would prefer just removing that comparison as any non-zero value will match anyway.

Reply:
ack